packages/manager/src/managers/prismicRepository/PrismicRepositoryManager.ts

```diff
@@ -134,7 +134,12 @@ export class PrismicRepositoryManager extends BaseManager {
 			`./app/dashboard/repositories/${args.domain}/exists`,
 			API_ENDPOINTS.PrismicWroom,
 		);
-		const res = await this._fetch({ url });
+		const res = await fetch(url.toString(), {
```

Comment:
We should keep using this._fetch, no?

Reply:
Could be pass a parameter that prevent authent on this._fetch?
I'm afraid if we do something to all fetch we will forget this one in the future

Reply:
You could add the same condition, if no cookies just don't pass it actually